django/db/migrations/serializer.py

```diff
@@ -67,7 +67,12 @@ def serialize(self):
         imports = ["import datetime"]
         if self.value.tzinfo is not None:
             imports.append("from django.utils.timezone import utc")
-        return repr(self.value).replace('<UTC>', 'utc'), set(imports)
+        # Serialize, replacing tzinfo implementation details:
+        serialized = (
+            repr(self.value)
+            .replace('datetime.timezone.utc', 'utc')
+        )
+        return serialized, set(imports)
```

Comment:
Can we restore the previous "style"?
```suggestion
        return repr(self.value).replace('datetime.timezone.utc', 'utc'), set(imports)
```